src/controller/handler/node.go

```diff
@@ -29,7 +29,6 @@ func PutNode(w http.ResponseWriter, r *http.Request) {
     in := struct {
         Name        string `json:"name"`
         Description string `json:"description"`
-        Status      string `json:"status"`
```

Comment:
thus `status` merged into `description`? just mention here is that: filter nodes via status will make the process more complex. If we would like to have better performance, extracting `status `from `description` again.